### PR TITLE
fix some style error using gofmt

### PR DESCRIPTION
**Pull request by max8899**
**What this PR does / why we need it**:
fix some style error using gofmt

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: third_party/forked/golang/expansion/expand_test.go [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package expansion
 
 import (
+	api "k8s.io/kubernetes/pkg/apis/core"
```
Comment (ceshihao) [inline third_party/forked/golang/expansion/expand_test.go]: It is recommended that imports are organized in groups

https://github.com/golang/go/wiki/CodeReviewComments#imports

**Review by ceshihao — CHANGES_REQUESTED** [review]

File: third_party/forked/golang/expansion/expand_test.go (hunk 1 above)
Comment (max8899) [reply third_party/forked/golang/expansion/expand_test.go]: @ceshihao fix this by using goimports

**Comment by idealhack** [comment]
@max8899 Thanks for your work. Would you please squash the commits into one?

**Review by idealhack — APPROVED** [review]

**Comment by max8899** [comment]
/test pull-kubernetes-bazel-build
--

**Pull request merged**